darwin/dataset/remote_dataset_v2.py

```diff
@@ -832,3 +856,18 @@ def register_multi_slotted(
                 print(f"  - {item}")
         print(f"Reistration complete. Check your items in the dataset: {self.slug}")
         return results
+
+    @retry(
+        wait=wait_exponential_jitter(initial=60, max=120),
+        stop=stop_after_attempt(5),
+        retry=retry_if_exception_type(requests.exceptions.RequestException),
```

Comment:
just to verify: isn't `RequestException` thrown by any other client condition than the one you coded?